### PR TITLE
Refacrot fuel py3

### PR DESCRIPTION
**Pull request by jurkov**
I want to close issue #117 with this pull request.

### CODE REVIEW

File: bytebot_config.py.example [hunk 1]
```diff
@@ -82,6 +82,9 @@ BYTEBOT_PLUGIN_CONFIG = {
     },
     'fuel': {
         'apikey': 'thiswoudbeyourapikey',
+            'lat': '50.9827792',
+            'lng': '11.0394426',
+            'rad': '10',
```
Comment (mape2k) [inline bytebot_config.py.example]: Wrong indentation

File: plugins/fuel.py [hunk 2]
```diff
@@ -1,102 +1,156 @@
...
+
+                cmd = '!'
+                bot.privmsg(target, '( ͡° ͜ʖ ͡°)')
+                bot.privmsg(target, 'Example commands:')
+                bot.privmsg(target, cmd + 'fuel')
+                bot.privmsg(target, cmd + 'fuel help')
+                bot.privmsg(target, cmd + 'fuel sort <fuel>')
+                bot.privmsg(target, cmd + 'fuel <place>')
+                bot.privmsg(target, cmd + 'fuel <place> sort <fuel>')
+
```
Comment (mape2k) [inline plugins/fuel.py]: Which values for fuel are allowed?

**Comment by jurkov** [comment]
I used autopep8 to fix pep8 issues. I tried to fix the other issues too. Please give some feedback.

**Comment by mape2k** [comment]
Everything seems to be ok. I can't test the plugin without apikey.
If there are any issues left we should just open a new ticket.

**Pull request merged**